connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSinkTaskThreadedTest.java

```diff
@@ -147,7 +150,6 @@ public void testPollsInBackground() throws Exception {
             workerTask.iteration();
         }
         workerTask.stop();
-        workerTask.awaitStop(Long.MAX_VALUE);
```

Comment:
Yup, seems fine, just worth pointing out. I would rather not continue to add changes to this PR anyway, it's becoming difficult to review due to the size.